pam/generate.sh

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
 set -euo pipefail
+set -x
```

Comment:
This shouldn't be enabled by default. I think it's a good line to have, but it should be commented by default and we can enable it if debugging becomes necessary.

Reply:
Yeah, was a leftover (as when building a package you DO want to see what's get launched).

Although given that `go generate` si pretty silent by default it wasn't too bad IMHO.